lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -550,6 +552,58 @@ def destroy(self, request, thread_id):
         return Response(status=204)
 
 
+class LearnerThreadView(APIView):
+    """
+    **Use Cases**
+
+        Fetch user's active threads
+
+    **Example Requests**:
+
+        GET /api/discussion/v1/courses/course-v1:ExampleX+Subject101+2015/learner/?page=1&page_size=10
+
+    **GET Thread List Parameters**:
+
+        * page: The (1-indexed) page to retrieve (default is 1)
+
+        * page_size: The number of items per page (default is 10)
+    """
+
+    def get(self, request, course_id=None):
+        """
+        Implements the GET method as described in the class docstring.
+        """
+        course_key = CourseKey.from_string(course_id)
+        page_num = request.GET.get('page', 1)
+        threads_per_page = request.GET.get('page_size', 10)
+        discussion_id = None
+        user_id = request.user.id
+        group_id = None
+        try:
+            group_id = utils.get_group_id_for_comments_service(request, course_key, discussion_id)
```

Comment:
Import the method and call only the method!
`from lms.djangoapps.discussion.django_comment_client.utils import get_group_id_for_comments_service`

`get_group_id_for_comments_service(request, course_key, discussion_id)`